test/test_manifest.json

```diff
@@ -60,6 +60,14 @@
        "enhance": true,
        "type": "text"
     },
+    {  "id": "tracemonkey-forced-colors-eq",
+      "file": "pdfs/tracemonkey.pdf",
+      "md5": "9a192d8b1a7dc652a19835f6f08098bd",
+      "rounds": 1,
+      "pageForegroundColor": "#00FF00",
+      "pageBackgroundColor": "black",
```

Comment:
Can we please format this as follow instead, which also simplifies the `test/driver.js` file a tiny bit?
```javascript
"pageColors": {
  "background": "black",
  "foreground": "#00FF00",
},
```